src/index.ts

```diff
@@ -304,17 +304,35 @@ export default class FlatfileImporter extends EventEmitter {
           return this.$validatorCallback ? this.$validatorCallback(row) : undefined
         },
         dataHookCallback: (row, index, mode) => {
-          return this.$recordHook ? this.$recordHook(row, index, mode) : undefined
+          try {
+            return this.$recordHook ? this.$recordHook(row, index, mode) : undefined
+          }catch ({message}) {
+            console.error(`Flatfile Record Hook Error: ${message}`)
```

Comment:
I think it would make sense to log the row, index, mode, and error stack

```suggestion
          }catch ({message, stack}) {
            console.error(`Flatfile Record Hook Error on row ${index}: ${message}`, stack, { row, mode })
```